packages/app/components/MemberCard/RoundedIcon.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Image, View } from "react-native";
+
+const RoundedIcon = ({ image }) => {
```

Comment:
I like to use the props parameter like in the link below:
https://reactnative.dev/docs/props